packages/lit-dev-content/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "wireit": {
     "build": {
-      "command": "LITDEV_ENV=local NODE_OPTIONS=--experimental-vm-modules eleventy",
+      "command": "LITDEV_ENV=local NODE_OPTIONS=\"--experimental-vm-modules --max_old_space_size=4096\" eleventy",
```

Comment:
since we're here, maybe we can move the `NODE_OPTIONS` env var setting to wireit's "env"? https://github.com/google/wireit#environment-variables

there might be other things preventing it from working, but this makes these scripts more Windows friendly.

Reply:
Great suggestion! TIL!